jscomp/test/js_string_test.js

```diff
@@ -29,7 +29,7 @@ function eq(loc, _z) {
 
 eq('File "js_string_test.ml", line 13, characters 5-12', /* tuple */[
       "012",
-      "0".concat("1", "2")
+      "0".splice("1", "2")
```

Comment:
Sorry, that's been fixed, just missed commiting the generated js. I'll add more tests to avoid issues like this.